### PR TITLE
Various bugfixes

### PR DESCRIPTION
**Pull request by dominikl**
Fixed bugs and how to test them:
- [Ticket 13137](http://trac.openmicroscopy.org/ome/ticket/13137) : Hardly reproducible, maybe code review of 32b359f is sufficient.
- [Ticket 13141](http://trac.openmicroscopy.org/ome/ticket/13141) : Repeat the workflow which triggered the bug, see first comment on the ticket.
- [QA 16976](https://www.openmicroscopy.org/qa2/qa/feedback/16976/) : Move the separator which separates the left hand side tree panel (while showing the project/datasets tree) and the center panel as far too the left as possible.
- [Ticket 13069](http://trac.openmicroscopy.org/ome/ticket/13069) : Also hardly reproducible, code review only, see c28c76f

### CODE REVIEW

**Comment by gusferguson** [comment]
@dominikl 

Tested using OMERO.insight-5.2.1-271-126effd-ice35-b210-win eel user-3 read-only-1

Ticket 13141 : Repeat the workflow which triggered the bug, see first comment on the ticket.
Behaves as expected - no crashes.

QA 16976 : Move the separator which separates the left hand side tree panel (while showing the project/datasets tree) and the center panel as far too the left as possible.
Behaves as expected - no crash.

Good to merge.

**Comment by dominikl** [comment]
Thanks @gusferguson . I'll just add an integration test for c28c76f . No re-review necessary for that, just see if the merge-integration-java job passes tomorrow.

File: components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java [hunk 1]
```diff
@@ -280,7 +280,7 @@ void setStartValue(double startValue)
 
 	/**
 	 * Sets the value of the end knob, value in the range
-	 * <code>]{@link #startValue}, {@link #maximum}]</code>.
+	 * <code>]{@link #startValue}, {@link #maximum}]  (one based) </code>.
```
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java]: open with a `[`?

Comment (dominikl) [inline components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java]: I assumed it was deliberate (range not including the startValue).

Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java]: It turns out you're right, it's a French notation I'd never seen before: http://hsm.stackexchange.com/questions/142/why-is-american-and-french-notation-different-for-open-intervals-x-y-vs-x so it at least has precedent. :+1:

Comment (dominikl) [inline components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java]: I guess it's an US-Europe thing, at least Germany typically also uses the square bracket notation.

Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java]: (I'd not seen in the UK either but, yes, approximately. :smiley:)

Comment (jburel) [inline components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java]: That's the way to indicate that the value is excluded (maths sense)

**Comment by jburel** [comment]
not related to this PRs but method like ```getProjects``` could include a check if ids.size = 0  (so it returns quickly) and if exp =-1
better to be done in another PR with some tests.

**Comment by dominikl** [comment]
Added it to the [Gateway board](https://trello.com/b/Tb2WuoBZ/gateway-api)

File: components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java (hunk 1 above)
Comment (mtbc) [inline components/insight/SRC/org/openmicroscopy/shoola/util/ui/slider/TwoKnobsSliderModel.java]: @jburel: I'm used to the http://mathworld.wolfram.com/Interval.html notation but that might just be coincidence with what I happen to have been exposed to!

**Comment by jburel** [comment]
@dominikl: thanks. Merging

**Pull request merged**